src/controllers/accounts/accounts.ts

```diff
@@ -43,6 +49,9 @@ export class AccountsController extends EventEmitter {
   // Holds the initial load promise, so that one can wait until it completes
   initialLoadPromise: Promise<void>
 
+  // all SignedMessage type 7702-authorization the user has signed
+  #authorizations: InternalSignedMessages
```

Comment:
The above 7702 comment is great, but the first time I saw #authorizations in the code, I thought we stored all types of auth messages there.